model/src/commonMain/kotlin/SessionType.kt

```diff
@@ -14,4 +14,10 @@ enum class SessionType(val id: String) {
     companion object {
         fun of(id: String?) = values().find { it.id == id } ?: Unknown
     }
+
+    val supportDetail: Boolean
```

Comment:
Can you add the main constructor property?
 `SessionType(val id: String, val supportDetail: Boolean)`
Because when add SessionType, we forget that property 💭